src/main/java/com/atlassian/jira/cloud/jenkins/deploymentinfo/pipeline/JiraSendDeploymentInfoStep.java

```diff
@@ -40,8 +45,9 @@
 
     @DataBoundConstructor
     public JiraSendDeploymentInfoStep(
```

Comment:
Only mandatory parameters can be in the constructor. so `site` needs to be removed and have only the setter.